### PR TITLE
Add a few tests for BoundedSet

### PR DESCRIPTION
**Pull request by PythonCoderAS**
This adds in some basic tests for the `__contains__` method, as well as making sure that the removal of items works. Finally, it tests the LRU property of the BoundedSet.

### CODE REVIEW

File: tests/unit/models/test_util.py [hunk 1]
```diff
@@ -40,6 +45,27 @@ def test_exponential_counter__reset(self):
...
+        assert len(bset._set) == 10
+        assert 0 not in bset
+
+    def test_lru(self):
+        bset = BoundedSet(max_items=10)
+        [bset.add(i) for i in range(10)]
+        assert 0 in bset
+        bset.add(0)
+        bset.add(10)
+        assert 0 in bset
```
Comment (leviroth) [inline tests/unit/models/test_util.py:65]: I think we need to assert that `1` is no longer in the set - that's the least recently used element.

File: tests/unit/models/test_util.py [hunk 2]
```diff
@@ -40,6 +45,27 @@ def test_exponential_counter__reset(self):
...
+        bset.add(1)
+        assert 1 in bset
+
+    def test_bound(self):
+        bset = BoundedSet(max_items=10)
+        [bset.add(i) for i in range(11)]
+        assert len(bset._set) == 10
+        assert 0 not in bset
+
+    def test_lru(self):
```
Comment (leviroth) [inline tests/unit/models/test_util.py]: Perhaps we should have two versions of this test: the current version shows that `add` marks the element as recently used, but it's also important that `__contains__` does.

**Review by leviroth — CHANGES_REQUESTED** [review]
I suggested some tweaks, but looks good otherwise.

In terms of praw's git history, would you like me to just merge this into my branch like a normal PR once it's all set?

File: tests/unit/models/test_util.py (hunk 1 above)
Comment (PythonCoderAS) [reply tests/unit/models/test_util.py:65]: I mean we can assert both.

**Comment by PythonCoderAS** [comment]
> I suggested some tweaks, but looks good otherwise.
> 
> In terms of praw's git history, would you like me to just merge this into my branch like a normal PR once it's all set?

You're welcome to merge it in as a normal PR.

**Review by leviroth — APPROVED** [review]

**Pull request merged**